docs/source/en/modular_transformers.md

```diff
@@ -43,6 +46,8 @@ be moved to the new Modular Transformers format in the coming months.
 
 ### Details 
 
+To generate the single-files from the modular file, run `python utils/modular_model_converter.py --files-to-parse src/transformers/models/<your_model>/modular_<your_model>.py`.
```

Comment:
Nicer to put the command in a copyable code cell :)

```suggestion
To generate a single file from the modular file, run the following command.

```bash
python utils/modular_model_converter.py --files-to-parse src/transformers/models/<your_model>/modular_<your_model>.py
```
```

Reply:
Yes, thanks, this is better 👍